portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,16 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function guessMyTrip()
+{
+  const places=["thailand", "singapore", "tokyo", "australia","peru", "hawaii", "vietnam"];
+  var input =document.getElementById("user_input").value;
+  console.log(input);
```

Comment:
Is this to debug or do you want this to print every time?

Reply:
Just for debug,  I will remove it now.